datashare-app/src/main/java/org/icij/datashare/com/mail/MailSender.java

```diff
@@ -23,15 +21,29 @@ public class MailSender {
     protected final Log logger = LogFactory.getLog(this.getClass());
     
     public final int port;
+    final String user;
+    final String password;
     public final String host;
 
     public MailSender(String host, int port) {
+        this(host, port, null, null);
+    }
+
+    public MailSender(URI uri) {
+        this(
+            uri.getHost(),
+            uri.getPort(),
+            uri.getUserInfo().split(":")[0],
```

Comment:
Do we need to check if it's empty or it will never happen or it is checked by the lib? (empty username or password)